idaes/power_generation/unit_models/boiler_heat_exchanger.py

```diff
@@ -163,21 +134,27 @@ class BoilerHeatExchangerData(UnitModelBlockData):
     """
 
     CONFIG = ConfigBlock()
```

Comment:
Minor comment, but why not use `CONFIG = UnitModelBlockData.CONFIG()` (or just rely on inheritance directly)? As far as I can tell, you are just re-declaring the contents of the base classes Config block anyway (dynamic and has_holdup).

Reply:
Will be fixed in a PR coming in a day or two I promise.